lib/collection/proxy-config.js

```diff
@@ -85,9 +117,23 @@ _.assign(ProxyConfig.prototype, /** @lends ProxyConfig.prototype */ {
         if (!_.isObject(options)) {
             return;
         }
-        _.has(options, 'server') && this.server.update(options.server);
-        _.isString(options.match) && (this.match = new UrlMatchPattern(options.match));
-        _.has(options, 'server') && (this.tunnel = _.isBoolean(options.tunnel) ? options.tunnel : false);
+        _.isString(options.host) && (this.host = options.host.replace(regexes.protocolMatcher, ''));
```

Comment:
If you're doing this, why are you assigning a default value separately above? Check some other constructors like `Url` etc to see how to do this right.